source/our_tools/tractoflow.rst

```diff
@@ -3,4 +3,220 @@
+
+       If it fails:
+
+         - Is the data binding correctly (see point 5)?
+         - If one process fails, you should see a warning such as ``[11/53e26e] NOTE: Process `Bet_Prelim_DWI (101309)` terminated with an error exit status (127) -- Execution is retried (3)``. You can then check in the associated folder's log to see the error. For example:
+
+         .. code-block:: bash
+
+            ls -a work/11/53e26e*/  # Check that files are there
+            cat work/11/53e26e*/.command.log  # Check the error
+
+    #. Last decision to take on the cluster: Do you need 1 node or multiple nodes? Beyond 20 subjects or so, we recommend using multiple nodes. See the HPC part of tractoflow's `launch page <https://tractoflow-documentation.readthedocs.io/en/latest/pipeline/launch.html>`_. Depending on the cluster you are using (beluga, cedar, etc.), you have access to different types of nodes. Keeping the option "mem=0" in your sbatch (see next step) means you accept any node. If your data is very heavy (ex, HCP data), you might need to restric to the nodes with more RAM. See the `memory section here <https://docs.computecanada.ca/wiki/Running_jobs#Memory>`_ for more information.
+
+    #. We have all the ingredients to prepare the final sbatch. Here, we ask for 4 nodes, with 32 threads each and 116Gb of RAM each (see the -with-mpi option). Create a file cmd_my_tractoflow.sh with the following.
+
+        .. code-block:: bash
+
+            #!/bin/sh
+            #SBATCH --mail-user=YOUR_EMAIL
+            #SBATCH --mail-type=BEGIN
+            #SBATCH --mail-type=END
+            #SBATCH --mail-type=FAIL
+            #SBATCH --mail-type=REQUEUE
+            #SBATCH --mail-type=ALL
+
+            #SBATCH --nodes=2
+            #SBATCH --cpus-per-task=32
+            #SBATCH --mem=0
+            #SBATCH --time=24:00:00
+
+            export NXF_CLUSTER_SEED=$(shuf -i 0-16777216 -n 1)
+
+            srun (copy your nextflow command from point 9) -with-mpi
+
+    #. Finally launch your sbatch! Yeah!
+
+        .. code-block:: bash
+
+            sbatch -A def-descotea cmd_my_tractoflow.sh
+
+            squeue -u USER  # To check if it has been launched
+
+            exit # To quit beluga
+
+        You will receive an email when your command is launched (if you provided it in step 11). If you want to supervise the results while it runs, you should find a file such as slurm-6635828.out. It is the equivalent of what would be printed in your terminal if you ran it directly. You can download it with scp or simply look at it using 'cat slurm-6635828.out'.
+
+        .. note:: It is normal to see some failed tasks. The way it works, many jobs are started at the same time, and it may cause some scheduling errors. As long as some jobs succeed, you can let it run.
+
+    #. We recommend doing something like this to save results, scripts and container
```

Comment:
It demands a lot. I personnally don't do that. I think it is not useful because anyway you are the only person that can see your scratch.

Reply:
We maybe must to see with Arnaud to move back the results etc on braindata with BIDS strucutre. @arnaudbore

Reply:
@mdesco This was your proposition for setting up. Any comment?

Reply:
Kept but I added a line to explain it should not be kept in the scratch. We can see for BIDS later.